plugins/spu2-x/src/Windows/SndOut_XAudio2.inl

```diff
@@ -16,42 +16,37 @@
  */
 
 #include <atlcomcli.h>
+#include <sstream>
+#include <stdexcept>
+#include <string>
 
 namespace Exception
 {
 class XAudio2Error : public std::runtime_error
 {
-protected:
-    static const char *SomeKindaErrorString(HRESULT hr)
+private:
```

Comment:
I suppose so. I could probably remove it (or swap the order and have the public stuff first).